Deliverables/R1 - April 16.md

```diff
@@ -0,0 +1,15 @@
+## 1. What have you accomplished this week? (Please list each accomplishment, and explain)
+
+This week I was able to add a Swanky new animation to know breaking process thanks to Conair reminding me that linear interpolation is a thing. I was also able to create backstage menus for my inventory screen, my overview screen, and my structures screen as outlined in my original GitHub issue. The inventory screen is now a much cleaner and scalable solution to handling inventory in the tracking of materials than what I was previously using as it contains all that information within a compound data structure as opposed to a collection of global variables. Additionally, while I don't have a measurable goal to display this, I also learned how GML handles data structures, variable scoping, enumeration, and partial Sprites which greatly streamlines and simplifies my workflow moving forward.
+
+## 2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+
+That leads into the main challenges and difficulties that I faced this week of not initially knowing how GML handled complex data structures, I did eventually get through it thanks to a lot of research, yo-yo's documentation, and several helpful YouTube videos. Though I am still a little bit confused on when I need to be using instance vs local variables, but I imagine that's the kind of thing I'll figure out with more experience.
```

Comment:
I definitely agree with this. Some ideas and issues that you have been facing are very much tool-related; I'm sure things such as these will be easier with time.